packages/colorpickers/stories/examples/ColorSwatch/Controlled.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Story, Meta } from '@storybook/react';
+import { ColorSwatch } from '@zendeskgarden/react-colorpickers';
+import { convertToMatrix } from '@zendeskgarden/container-utilities';
+
+export default {
+  title: 'Components/ColorSwatch',
+  component: ColorSwatch
+} as Meta;
+
+export const colors = [
+  { label: 'Green-800', value: '#0b3b29' },
+  { label: 'Green-700', value: '#186146' },
+  { label: 'Green-600', value: '#038153' },
+  { label: 'Green-500', value: '#228f67' },
+  { label: 'Green-400', value: '#5eae91' },
+  { label: 'Green-300', value: '#aecfc2' },
+  { label: 'Green-200', value: '#d1e8df' },
+  { label: 'Red-800', value: '#681219' },
+  { label: 'Red-700', value: '#8c232c' },
+  { label: 'Red-600', value: '#cc3340' },
+  { label: 'Red-500', value: '#d93f4c' },
+  { label: 'Red-400', value: '#e35b66' },
+  { label: 'Red-300', value: '#f5b5ba' },
+  { label: 'Red-200', value: '#f5d5d8' },
+  { label: 'Blue-800', value: '#0f3554' },
+  { label: 'Blue-700', value: '#144a75' },
+  { label: 'Blue-600', value: '#1f73b7' },
+  { label: 'Blue-500', value: '#337fbd' },
+  { label: 'Blue-400', value: '#5293c7' },
+  { label: 'Blue-300', value: '#adcce4' },
+  { label: 'Blue-200', value: '#cee2f2' },
+  { label: 'Yellow-800', value: '#703b15' },
+  { label: 'Yellow-700', value: '#ad5e18' },
+  { label: 'Yellow-600', value: '#ed961c' },
+  { label: 'Yellow-500', value: '#f5a133' },
+  { label: 'Yellow-400', value: '#ffb648' },
+  { label: 'Yellow-300', value: '#fcdba9' },
+  { label: 'Yellow-200', value: '#fff0db' }
+];
```

Comment:
It would be good to use Garden's [PALETTE](https://garden.zendesk.com/components/palette#palette) to automatically generate these label-value pairs. Consider https://github.com/zendeskgarden/website/blob/c59d385187fccae70954b85c771027947aaf2067/src/components/MarkdownProvider/components/ColorPalette.tsx#L88-L97.

Reply:
Extract that `reduce` to a utility function for reuse among all packages?

Reply:
Hmm, I took a look at `ColorPalette` and don't feel like deriving this data from `PALETTE` is worth writing a function for. If all the colors are used (it isn't in the Figma assets) then I think there's a case to be made for it to simply run through a reducer function. However, I'm inclined not to spend a significant amount of time writing logic to parse out `PALETTE` if there isn't a large pain point here. 

But maybe I'm overthinking this. How did you envision the function to look like @jzempel?

Reply:
>  don't feel like deriving this data from `PALETT`E is worth writing a function for

I envisioned a different variation on the `reduce` that I linked to in my original comment. This is used to produce the output for https://garden.zendesk.com/design/color (which is essentially finessed name-hex color pairs). Your version would only consider palette shades 200-800 across a limited set of hues. And limited to storybook. Just looking for example code that adapts to future Garden palette changes – I think the minimal extra effort is worth it.